installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: ASSEMBLY
+[id="installing-nutanix-installer-provisioned"]
+= Installing a cluster on Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: installing-nutanix-installer-provisioned
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on your Nutanix instance using installer-provisioned infrastructure.
+
+== Prerequisites
+
+* You have reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* If you use a firewall, you have configured it to xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[grant access] to the sites that {product-title} requires. This includes the use of Telemetry.
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/installation-initializing.adoc[leveloffset=+1]
+include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
+include::modules/installation-nutanix-config-yaml.adoc[leveloffset=+2]
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
+include::modules/manually-configure-iam-nutanix.adoc[leveloffset=+1]
+
+include::modules/installation-launching-installer.adoc[leveloffset=+1]
+
+== Configuring the default storage container
+
+After you install the cluster, you must configure the default storage container for the cluster.
+
+<TBD>
+
```

Comment:
We need to cover installing the default storage container after installing the cluster. Are these documented somewhere or should I schedule some time to step through them?

Reply:
The current plan is to use the existing documentation [here](https://github.com/nutanix/openshift/blob/main/operators/csi/README.md). We also need to include documentation on configuring registry storage, which should be published [here](https://github.com/nutanix/openshift/blob/main/docs/install/ipi/README.md). I also think we should include the CLI install doc before these two (like in current vSphere doc).

Reply:
@makentenza We don't plan to publish this doc on our traditional portal. If you're against GitHub doc, then we have another option we're working on. We can discuss offline.

Reply:
@mjpytlak Links are [CSI install doc](https://opendocs.nutanix.com/openshift/operators/csi/) and [registry storage config](https://opendocs.nutanix.com/openshift/install/ipi/#openshift-image-registry-configuration).

Reply:
Added to doc. Thank you.